src/VisualStudio/Xaml/Impl/Implementation/XamlTextViewCreationListener_IVsRunningDocTableEvents3.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.LanguageServices.Xaml
+{
+    internal partial class XamlTextViewCreationListener : IVsRunningDocTableEvents3
+    {
+        public int OnAfterAttributeChange(uint docCookie, uint grfAttribs)
+        {
+            return VSConstants.S_OK;
```

Comment:
I believe VSConstants.E_NOTIMPL is recommended in these, as some callers know you don't care and can avoid future calls.